### PR TITLE
add customer phone + remove deleted-customer filter 

### PR DESCRIPTION
**Pull request by fivetran-jamie**
## PR Overview
**This PR will address the following Issue/Feature:**
- https://github.com/fivetran/dbt_stripe_source/issues/71
- https://github.com/fivetran/dbt_stripe_source/issues/70

**This PR will result in the following new package version:**

v0.12.0 because we are changing the output of the customer_overview model -- deleted customers are now included

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt compile
- [x] dbt run –full-refresh
- [x] dbt run
- [x] dbt test
- [ ] dbt run –vars (if applicable)

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked and tagged
- [x] You are assigned to the corresponding issue and this PR
- [x] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [x] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [x] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [x] You have provided details below around the validation steps performed to gain confidence in these changes.

see height task

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [x] indexed within dbt_project.yml
    - [x] indexed within integration_tests/dbt_project.yml
- [x] CHANGELOG has individual entries for each respective change in this PR
    
- [x] README updates have been applied (if applicable)
    
- [ ] DECISIONLOG updates have been updated (if applicable)
- [x] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [ ] docs were regenerated (unless this PR does not include any code or yml updates) -- WILL REGEN AFTER APPROVAL

### If you had to summarize this PR in an emoji, which would it be?
 
📞

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# dbt_stripe v0.12.0
+[PR #75](https://github.com/fivetran/dbt_stripe/pull/75) includes the following updates:
+
+## Feature Updates
+- Adds the `phone` column to `stripe__customer_overview`. 
+- No longer filters out deleted customers in `stripe__customer_overview`.
+  - Persists `is_deleted` field to differentiate between deleted and active customers.
+  - Note that this is a 🚨 breaking change 🚨, as previously filtered-out records will appear in `stripe__customer_overview`.
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:8]: Same comment from the source where I would prefer we make this it's own section at the top to really highlight this.

File: packages.yml [hunk 2]
```diff
@@ -1,3 +1,7 @@
 packages:
-- package: fivetran/stripe_source
-  version: [">=0.10.0", "<0.11.0"]
+# - package: fivetran/stripe_source
+#   version: [">=0.10.0", "<0.11.0"]
+# - local: ../../Finance/Stripe/dbt_stripe_source
+- git: https://github.com/fivetran/dbt_stripe_source.git
+  revision: update/customer360
+  warn-unpinned: false
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:7]: Reminder to swap before release

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM with one small comment about updating the CHANGELOG

File: CHANGELOG.md (hunk 1 above)
Comment (fivetran-jamie) [reply CHANGELOG.md:7]: updated

**Review by fivetran-catfritz — APPROVED** [review]
looks good for release!

**Pull request merged**